projects/packages/search/src/widgets/class-search-widget.php

```diff
@@ -976,6 +983,9 @@ public function render_widget_edit_filter( $filter, $is_template = false, $is_in
 						<option value="author" <?php $this->render_widget_option_selected( 'type', $args['type'], 'author', $is_template ); ?>>
 							<?php esc_html_e( 'Author', 'jetpack-search-pkg' ); ?>
 						</option>
+						<option value="blog_id" <?php $this->render_widget_option_selected( 'type', $args['type'], 'blog_id', $is_template ); ?>>
+							<?php esc_html_e( 'Blog ID', 'jetpack-search-pkg' ); ?>
```

Comment:
I think displaying `Blog` as a filter name rather than `Blod ID` would be more meaningful to users.

Reply:
@dognose24 Thank you for the code review, I have renamed it.